lib/usecases/auth_token_storage_use_case.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter_survey/model/login_model.dart';
+import 'package:flutter_survey/usecases/base/base_use_case.dart';
+import 'package:flutter_survey/database/secure_storage.dart';
+import 'package:injectable/injectable.dart';
+
+@Injectable()
+class AuthTokenStorageUseCase extends DataSaverUseCase<LoginModel> {
```

Comment:
A use case is an action so its name should start with a verb.

Reply:
Agree, Ex: `StoreAuthTokenUseCase` 🚀